app/config/site-templates.php

```diff
@@ -2,60 +2,37 @@
 
 const TEMPLATE_FRAMEWORKS = [
     'SVELTEKIT' => [
-        'name' => 'sveltekit'
-    ],
-    'NEXTJS' => [
-        'name' => 'nextjs'
+        'name' => 'Svelte Kit',
+        'installCommand' => 'npm install',
```

Comment:
tiny nitpick, lets keep in mind to use readonly actions like `npm ci` for when its going to prod 👍🏻